pkg/interface/src/views/apps/links/components/LinkItem.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState }  from 'react';
+import { Link } from 'react-router-dom';
+import { Row, Col, Anchor, Box, Text, BaseImage, Icon, Action } from '@tlon/indigo-react';
+
+import { Sigil } from '~/logic/lib/sigil';
+import { writeText } from '~/logic/lib/util';
+import Author from '~/views/components/Author';
+
+import { roleForShip } from '~/logic/lib/group';
+import { Contacts, GraphNode, Group, LocalUpdateRemoteContentPolicy, Rolodex } from '~/types';
+import GlobalApi from '~/logic/api/global';
+import { Dropdown } from '~/views/components/Dropdown';
+import RemoteContent from '~/views/components/RemoteContent';
+
+interface LinkItemProps {
+  node: GraphNode;
+  resource: string;
+  hideAvatars: boolean;
+  hideNicknames: boolean;
+  remoteContentPolicy: LocalUpdateRemoteContentPolicy;
+  api: GlobalApi;
+  group: Group;
+  contacts: Rolodex;
+}
+
+export const LinkItem = (props: LinkItemProps) => {
+  const {
+    node,
+    resource,
+    hideAvatars,
+    hideNicknames,
+    remoteContentPolicy,
+    api,
+    group,
+    contacts,
+    ...rest
+  } = props;
+
+  const URLparser = new RegExp(
+    /((?:([\w\d\.-]+)\:\/\/?){1}(?:(www)\.?){0,1}(((?:[\w\d-]+\.)*)([\w\d-]+\.[\w\d]+))){1}(?:\:(\d+)){0,1}((\/(?:(?:[^\/\s\?]+\/)*))(?:([^\?\/\s#]+?(?:.[^\?\s]+){0,1}){0,1}(?:\?([^\s#]+)){0,1})){0,1}(?:#([^#\s]+)){0,1}/
+  );
+
+  const author = node.post.author;
+  const index = node.post.index.split('/')[1];
+  const size = node.children ? node.children.size : 0;
+  const contents = node.post.contents;
+  const hostname = URLparser.exec(contents[1].url) ? URLparser.exec(contents[1].url)[4] : null;
+
+  const baseUrl = props.baseUrl || `/~404/${resource}`;
+
+  const ourRole = group ? roleForShip(group, window.ship) : undefined;
+  const [ship, name] = resource.split('/');
+
+  const [locationText, setLocationText] = useState('Copy Link Location');
+
+  const copyLocation = () => {
+    setLocationText('Copied');
+    writeText(contents[1].url);
+    setTimeout(() => {
+      setLocationText('Copy Link Location');
+    }, 2000);
+  };
+
+  const deleteLink = () => {
+    if (confirm('Are you sure you want to delete this link?')) {
+      api.graph.removeNodes(`~${ship}`, name, [node.post.index]);
+    }
+  };
+
+  return (
+    <Box width="100%" {...rest}>
+    
+      <Box
+        lineHeight="tall"
+        display='flex'
+        flexDirection='column'
+        width="100%"
+        color='washedGray'
+        border={1}
+        borderRadius={2}
+        alignItems="flex-start"
+        overflow="hidden"
+      >
+        <RemoteContent
+          url={contents[1].url}
+          text={contents[0].text}
+          remoteContentPolicy={remoteContentPolicy}
+          style={{ marginTop: '-1px' }}
```

Comment:
```suggestion
          style={{ marginTop: '-1px', alignSelf: 'center' }}
```

We want images to be centred in the item ... this suggestion however will also centre the title text as well. Any idea here?